### PR TITLE
fix: 590 - get rid of orphans

### LINKED ISSUE #590: Get rid of orphans
## 🐛 Describe the Bug

In typography orphans is a single word (or syllable) that sits at the bottom of a paragraph of text.

Image

Orphans looks odd, typically has poor readability, and they should be avoided if possible. 

if you want to read more, here is the in-depth article -https://fonts.google.com/knowledge/glossary/widows_orphans

## 🏁 Actual Behavior 

Our site is full of orphans

Image
Image

## 🏁 Expected Behavior

Remove all the orphans in a single line of css
- [ ] Add `text-wrap: pretty` to the `body` element

If you want to read more about `text-wrap: pretty` - https://developer.mozilla.org/en-US/docs/Web/CSS/text-wrap#pretty

### PR DESCRIPTION
**Pull request by saripovdenis**
## What type of PR is this? (select all that apply)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 🚧 Breaking Change
- [ ] 🧑‍💻 Code Refactor
- [ ] 📝 Documentation Update

## Description
Added `text-wrap: pretty` to get rid of orphans

## Related Tickets & Documents
- Closes #590 

## Screenshots, Recordings

_Please replace this line with any relevant images for UI changes._

## Added/updated tests?

- [ ] 👌 Yes
- [x] 🙅‍♂️ No, because they aren't needed
- [ ] 🙋‍♂️ No, because I need help

## Summary by CodeRabbit

- **New Features**
	- Introduced a new CSS property `text-wrap: pretty;` for improved text formatting in the application.
  
- **Style**
	- Updated styles in the main stylesheet to enhance text presentation while maintaining existing layout and formatting.

### CODE REVIEW

File: src/app/styles/index.scss [hunk 1]
```diff
@@ -6,6 +6,8 @@ html {
 
 body {
   font-family: $font-primary;
+  // https://github.com/rolling-scopes/site/issues/590
```
Comment (SpaNb4) [inline src/app/styles/index.scss:9]: I'd prefer not to leave a comment here, but let's wait for other team members' opinions

Comment (Quiddlee) [reply src/app/styles/index.scss:9]: i agree

**Review by Quiddlee — APPROVED** [review]

File: src/app/styles/index.scss (hunk 1 above)
Comment (saripovdenis) [reply src/app/styles/index.scss:9]: Okay, will change it

**Comment by ansivgit** [comment]
run visual now

File: src/app/styles/index.scss (hunk 1 above)
Comment (saripovdenis) [reply src/app/styles/index.scss]: Done 🥳

**Review by SpaNb4 — APPROVED** [review]

**Pull request merged**